docs/src/content/docs/fr/guides/authoring-content.md

```diff
@@ -107,15 +107,15 @@ Les titres de niveau 2 (`<h2>`) et de niveau 3 (`<h3>`) apparaissent automatique
 
 Pour en apprendre davantage sur la façon dont Astro traite les attributs `id` des titres de section, consultez la [documentation d'Astro](https://docs.astro.build/fr/guides/markdown-content/#identifiants-den-t%C3%AAte).
 
-## Asides
+## Encarts
 
-Les Asides (également connus sous le nom de "callouts") sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
+Les encarts (également connus sous le nom de "asides") sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
 
-Starlight fournit une syntaxe Markdown personnalisée pour le rendu des apartés. Les blocs d'apartés sont indiqués en utilisant une paire de triples points `:::` pour envelopper votre contenu, et peuvent être de type `note`, `tip`, `caution` ou `danger`.
+Starlight fournit une syntaxe Markdown personnalisée pour le rendu des encarts. Les blocs d'encarts sont indiqués en utilisant une paire de triples points `:::` pour envelopper votre contenu, et peuvent être de type `note`, `tip`, `caution` ou `danger`.
 
 Vous pouvez imbriquer n'importe quel autre type de contenu Markdown à l'intérieur d'un aparté, mais les aparté sont mieux adaptés à des morceaux de contenu courts et concis.
 
-### Note aside
+### Note d'encart
```

Comment:
```suggestion
### Encart de type note
```

The main focus should be the "encart" so let's use the same syntax you originally used a little bit below:

> Les apartés de type Attention et Danger sont utiles pour attirer […]